City-of-Helsinki/drupal-infofinland
# Feature/IFU-866 remove ptv data

# [IFU-866](https://helsinkisolutionoffice.atlassian.net/browse/IFU-866)
The PTV module was removed previously as it is no longer needed, but this time we removed the fields related to PTV, also. (And data)

## What was done
This PR has 2 separate commits. All PTV related fields were removed and config exported, but some parts of the export didn't seem like they were related (e.g. simple XML sitemap) so those were in the second commit to be more easily removed if they don't belong here.

This PR was based on the d10 upgrade PR. It is probably wise to check this AFTER the d10 upgrade is complete.

## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout feature/IFU-866-remove-ptv-data`
  * `make fresh`
* Run `make drush-cr`

## How to test
Login and poke around to see if there's any evidence of PTV related fields remaining on paragraphs, content types, etc. The field list under reports may also be helpful.

## Designers review


[IFU-866]: https://helsinkisolutionoffice.atlassian.net/browse/IFU-866?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
